src/client/Microsoft.Identity.Client/ManagedIdentity/AppServiceManagedIdentitySource.cs

```diff
@@ -13,7 +15,7 @@ namespace Microsoft.Identity.Client.ManagedIdentity
     internal class AppServiceManagedIdentitySource : AbstractManagedIdentity
     {
         // MSI Constants. Docs for MSI are available here https://docs.microsoft.com/azure/app-service/overview-managed-identity
-        private const string AppServiceMsiApiVersion = "2019-08-01";
+        private const string AppServiceMsiApiVersion = "2025-03-30";
```

Comment:
Does this version of app service endpoint the equivalent to MITS endpoint mentioned in the spec?

Reply:
Yes, it is the same as SF MITS. This new version is now available in Canary region (EUAP) for testing Token Revocation. Basically, it accepts the new xms_cc and the bad token hash params